README.md

```diff
@@ -1,6 +1,54 @@
 # BMX
 
-BMX grants you API access to your AWS accounts, based on Okta credentials that you already own.
-It uses your Okta identity to create short-term AWS STS tokens, as an alternative to long-term IAM access keys.
+BMX provides API access to your AWS accounts using existing Okta credentials. Instead of long-term IAM access keys, BMX creates short-term AWS STS tokens using your Okta identity.
 
-This is a work in progress dotnet 7 branch
+# Installation
```

Comment:
Since "# BMX" is the top level title, sections should use a lower level headings
```suggestion
## Installation
```

Similarly for sections below e.g. Commands